net/node/node.go

```diff
@@ -375,6 +376,22 @@ func (node *node) Xmit(message interface{}) error {
 	return nil
 }
 
+func (node *node) Relay(from Noder, txn *transaction.Transaction) error {
```

Comment:
Should we change the name Relay here? It's different from our Proof of Relay and may cause some confusions. What about BroadcastTransaction?

Reply:
Good idea, let me fix it.